docs/reference/query-dsl/minimum-should-match.asciidoc

```diff
@@ -1,13 +1,14 @@
 [[query-dsl-minimum-should-match]]
 == Minimum Should Match
 
+The property `minimum_should_match` can be used with a bool `should` that has multiple clauses. Since multiword `match` and `multi-match` queries simply wrap the generated term queries in a bool query, they can use `minimum_should_match` as well.
```

Comment:
There's a couple of other queries (e.g. More Like This, Common Terms, Query String Query) that support `minimum_should_match` and link here because they internally use bool queries. Mentioning "bool" queries so prominently here might raise more questions than it answers. Maybe this can be rephrased a bit to make it more general?